L1Trigger/L1TGlobal/plugins/L1TExtCondProducer.cc

```diff
@@ -82,8 +90,26 @@ using namespace l1t;
     setBptxAND_ (iConfig.getParameter<bool>("setBptxAND")),
     setBptxPlus_ (iConfig.getParameter<bool>("setBptxPlus")),
     setBptxMinus_ (iConfig.getParameter<bool>("setBptxMinus")),
-    setBptxOR_ (iConfig.getParameter<bool>("setBptxOR"))
+    setBptxOR_ (iConfig.getParameter<bool>("setBptxOR")),
+    tcdsInputTag_(iConfig.getParameter<edm::InputTag>("tcdsRecordLabel"))
   {
+
+    makeTriggerRulePrefireVetoBit_=false;
+
+    m_triggerRulePrefireVetoBit=255;
```

Comment:
@apana @rekovic this magic value looks related to https://github.com/cms-sw/cmssw/blob/master/DataFormats/L1TGlobal/interface/GlobalExtBlk.h#L47 and now everything looks consistent. But why having the two as independent, and using a LogWarning that risks to go unnoticed? If this has to be the last bit of the array just use that position no? Or I would escalate the severity to LogError at least (although still it does not seem the best solution to me...)